chainio/clients/avsregistry/writer.go

```diff
@@ -32,13 +33,9 @@ type AvsRegistryWriter interface {
 	// socket? RegisterOperatorInQuorumWithAVSRegistryCoordinator is used to register a single operator with the AVS's
 	// registry coordinator. - operatorEcdsaPrivateKey is the operator's ecdsa private key (used to sign a message to
 	// register operator in eigenlayer's delegation manager)
-	//  - operatorToAvsRegistrationSigSalt is a random salt used to prevent replay attacks
-	//  - operatorToAvsRegistrationSigExpiry is the expiry time of the signature
 	RegisterOperatorInQuorumWithAVSRegistryCoordinator(
 		ctx context.Context,
 		operatorEcdsaPrivateKey *ecdsa.PrivateKey,
-		operatorToAvsRegistrationSigSalt [32]byte,
```

Comment:
is the assumption here is that most users won't want to specify this and are okay using the random salt?

Reply:
yep